dlrover/trainer/torch/flash_checkpoint/megatron_dist_ckpt.py

```diff
@@ -38,8 +38,8 @@
         update_num_microbatches,
     )
     from megatron.core import mpu, tensor_parallel
-    from megatron.optimizer.optimizer import ChainedOptimizer
-    from megatron.utils import print_rank_0, unwrap_model
+    from megatron.core.optimizer.optimizer import ChainedOptimizer
+    from megatron.training.utils import print_rank_0, unwrap_model
```

Comment:
It better to retry import apis if an ImportError happens to keep backward compatibility.

```Python
from megatron.core.optimizer.optimizer import ChainedOptimizer
try:
    from megatron.training import get_args
    from megatron.training.checkpointing import (
        check_checkpoint_args,
        find_checkpoint_rank_0,
        fix_query_key_value_ordering,
        update_num_microbatches,
    )
    from megatron.core.optimizer.optimizer import ChainedOptimizer
    from megatron.training.utils import print_rank_0, unwrap_model
except ImportError:
   from megatron import get_args
   from megatron.checkpointing import (
        check_checkpoint_args,
        find_checkpoint_rank_0,
        fix_query_key_value_ordering,
        update_num_microbatches,
    )
   from megatron.optimizer.optimizer import ChainedOptimizer
   from megatron.utils import print_rank_0, unwrap_model
 

```

Reply:
sure,you are right